appveyor.yml

```diff
@@ -0,0 +1,31 @@
+version: '1.0.{build}'
+
+cache:
+  - vendor/bundle
+
+environment:
+  RAILS_ENV: test
+  PGHOST: 127.0.0.1
+  PGUSER: postgres
+  POSTGRES_DB: circuitverse_test
+  PGPASSWORD: Password12!
+services:
+  - postgresql95
+
+install:
+  - set PATH=C:\Ruby26\bin;%PATH%
+  - bundle config --local path vendor/bundle
+  - bundle install --with postgresql
+
+build: off
+
+before_test:
+  - ruby -v
+  - gem -v
+  - bundle -v
+  - copy config\database.example.yml config\database.yml
+  - ruby bin\rails db:setup
+
+
+test_script:
+  - bundle exec coveralls push
```

Comment:
I've enabled the appveyor plugin for this repository, we can add the `bundle exec rspec` command here and test if the builds are running successfully.

Reply:
`bundle exec coveralls push` also executes rspec

Reply:
Coveralls need to run after rspec command

Reply:
I believe it will ,their docs say,

> To do this with RSpec, just type bundle exec coveralls push in your project directory.
> 
> This will run RSpec and upload the coverage data to Coveralls as a one-off build

Reply:
Btw I have to say I tried running rspec directly on appveyor and it takes a lot of time(more than 10 mins)